R/T_and_F_symbol_linter.R

```diff
@@ -68,5 +66,5 @@ T_and_F_symbol_linter <- function() { # nolint: object_name.
       make_lints(bad_usage, "Use %s instead of the symbol %s."),
       make_lints(bad_assignment, "Don't use %2$s as a variable name, as it can break code relying on %2$s being %1$s.")
     )
-  })
+  }, linter_level = "expression")
```

Comment:
I'm not a fan of this style, WDYT about rearranging instead:

```r
Linter(linter_level = "expression", function(source_expression) {
  ...
})
```

Reply:
SGTM I'll handle this in the next pass.